app.py

```diff
@@ -96,6 +104,7 @@ def base_search(query: str, search_method: Callable[[Elasticsearch, str, str], N
         orderedLoad = lambda x: json.loads(x, object_pairs_hook=OrderedDict)
         result_value = list(map(orderedLoad, rd.execute_command("LRANGE", query, 0, -1)))
         return jsonify(result={
+            "id"
```

Comment:
If we already get the id in the response, is this addition would be unnecessary?

Reply:
Yeah this was a mistake, seems like I forgot to remove it.